src/test/java/by/issoft/tests/CartTest.java

```diff
@@ -0,0 +1,47 @@
+package by.issoft.tests;
+
+import by.issoft.pagefactory.CartPage;
+import by.issoft.pagefactory.LoginPage;
+import by.issoft.pagefactory.ProductPage;
+import by.issoft.pagefactory.SearchPage;
+import by.issoft.utils.MyTestWatcher;
+import by.issoft.utils.TestBase;
+import io.qameta.allure.Description;
+import io.qameta.allure.Step;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(MyTestWatcher.class)
+public class CartTest extends TestBase {
+
+    String firstProduct = "Faded Short Sleeve T-shirts",
+            secondProduct = "Blouse",
+            thirdProduct = "Printed Chiffon Dress";
+
+    @Test
+    @Step("User can successfully add 3 products to cart")
+    @Description("User can successfully add 3 products to cart")
+    void userShouldBeAbleToAddProductsToCart() {
+        LoginPage loginPage = new LoginPage(driver);
+        driver.get("http://automationpractice.com/index.php?controller=authentication&back=my-account");
```

Comment:
Looks like you are using the same URL in all tests. It sound like the same step(s) for all you tests. What about to create method in TestBase and call it before each test ?